service/src/test/resources/db/migration/V1475253311__session_create_other_extern_table.sql

```diff
@@ -0,0 +1,18 @@
+/***********************************************************************************************************************
+  File: V1475253311__session_create_other_extern_table.sql
+
+  Desc: Creates an _extern table.  This table stores some specific session extern data.  Existing table in the database
+
+***********************************************************************************************************************/
+USE session;
+
+DROP TABLE IF EXISTS _externs;
+
+CREATE TABLE _externs (
+  clientname varchar(100) NOT NULL,
+  shiftwindowstart int(11) NOT NULL DEFAULT 0,
+  shiftwindowend int(11) NOT NULL DEFAULT 0
+);
+
+INSERT INTO _externs VALUES ('SBAC', 5, 3);
```

Comment:
I don't mind this type of data being in this project's migration script.  The reason is that this is seed data that is only read and never modified in the microservice.